src/pages/TeacherDashboard/TeacherDashboard.jsx

```diff
@@ -1,21 +1,21 @@
-import React, {useState, useEffect} from 'react';
+import {useState, useEffect} from 'react';
 import styles from "./TeacherDashboardStyles.module.css"
 import SideBar from "../../components/SideBar/SideBar";
 import {useNavigate} from "react-router-dom";
 
 const TeacherDashboard = () => {
-    const apiUrl = import.meta.env.VITE_API_BASE_URL;
+    const baseUrl = import.meta.env.VITE_BASE_URL;
```

Comment:
I've just merged in a change that changes Laura's code to use VITE_API_BASE_URL, so let's stick with that instead of VITE_BASE_URL and be consistent everywhere. Apologies if I asked you to change it initially.